src/components/Onboarding/OnboardingCard.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+import { useState, useEffect } from "react";
+import NicknameInput from "@/components/Onboarding/NicknameInput";
+import UserNameInput from "./UserNameInput";
+import { PaperAirplaneIcon } from "@heroicons/react/24/outline";
+import { Metadata } from "next";
+import { Button } from "@mantine/core";
+import { useForm } from "@mantine/form";
+
+export const metadata: Metadata = {
+    title: "CommonGround - Onboarding",
+    keywords: "onboarding, user, registration",
+    description: "CommonGround onboarding page for new users",
```

Comment:
The description should be in Chinese as our users should mostly be based in Taiwan